packages/2d/src/components/Video.ts

```diff
@@ -263,11 +281,34 @@ export class Video extends Rect {
     }
   }
 
+  protected getPlaybackRate() {
+    return this.playbackRate.context.getter();
+  }
```

Comment:
I mentioned this to give you a complete picture but it's already the default behavior
Conceptually, it's similar to calling the `super` method:
```ts
protected getPlaybackRate() {
  return super.getPlaybackRate();
}
```
So the entire thing can be omitted.

Reply:
Yeah, that makes sense. It didn't work without a getter before, but that was obviously because I didn't call the default setter. Will get rid of this method.